docs/source/arguments.rst

```diff
@@ -304,3 +303,69 @@ dglke_partition
   - ``-k NUM_PARTS`` or ``--num-parts NUM_PARTS``
     The number of partitions.
 
+dglke_score
+^^^^^^^^^^^^^^^
+  - ``--data_path DATA_PATH``
+    The root path of all dataset including id mapping files. Default: 'data'
+
+  - ``--model_path MODEL_PATH``
+    The place where to load the model. Default 'ckpts'.
+
+  - ``--format FORMAT``
+    The format of the input data, specified in ``h_r_t``. Ideally, user should provides three files, one for head entities, one for relations and one for tail entities. But we also allow users to use *** to represent *all* of the entities or relations. For example, ``h_r_*`` requires users to provide files containing head entities and relation entities and use the whole entity set as tail entities; ``*_*_t`` requires users to provide a single file containing tail entities and use the whole entity set as head entities and the whole relation set as relations. The supported formats include ``h_r_t``, ``h_r_*``, ``h_*_t``, ``*_r_t``, ``h_*_*``, ``*_r_*``, ``*_*_t``. By default, the calculation will take an N_h x N_r x N_t manner.
+
+  - ``--data_files [DATA_FILES ...]`` 
+    A list of data file names. This is used to provide necessary files containing the requried data according to the format, e.g., for ``h_r_t``, three files are required as h_data, r_data and t_data, while for ``h_*_t``, two files are required as h_data and t_data.
+
+  - ``--raw_data``
+    A flag tells whether the data profiled in data_files is in the raw object naming space or in mapped id space. If True, the data is in the original naming space and the inference program will do the id translation according to id mapping files. If False, the data is just intergers and it is assumed that user has already done the id translation. Default: False.
+
+  - ``--bcast``
+    Whether to broadcast topK in a specific side. By default, an universal topK across all scores are returned. Users can specify ``head`` to broadcast at head that returns topK for each head; ``rel`` to broadcast at relation that returns topK for each relation; ``tail`` to broadcast at tail that returns topK for each tail. Default: None.
```

Comment:
don't quite understand this one either. I suppose this is bool, so Default is False?

Reply:
A string. if you have a list of h, r, t and want to calculate top5 most relevant links for each $h_i$, you can specify --bcast 'head'

Reply:
but h_r_t doesn't broadcast head, relation and tail? i think you have an option to do pair-wise computation, right? so i suppose the default behavior is broadcast?